backend/apis/clients/get-client.api.js

```diff
@@ -0,0 +1,150 @@
+const { app, databaseError, pool, invalidRequest } = require("../../server");
+const mysql = require("mysql");
+const { checkValid, validInteger } = require("../utils/validation-utils");
+const { responseFullName } = require("../utils/transform-utils");
+
+app.get("/api/clients/:id", (req, res, next) => {
```

Comment:
This API will come in handy when we're building the page for a single client where we show them all the client info and then let them modify any of it that they want to.